cli/azd/pkg/infra/provisioning/bicep/bicep_provider_test.go

```diff
@@ -974,7 +975,39 @@ func TestUserDefinedTypes(t *testing.T) {
 			"sku":  {Type: "string"},
 		},
 		objectParam.Properties)
+	require.NotNil(t, objectParam.AdditionalProperties)
+	require.Equal(
+		t,
+		azure.ArmTemplateParameterAdditionalProperties{
+			Type:      "string",
+			MinLength: to.Ptr(10),
+			Metadata: map[string]json.RawMessage{
+				"fromDefinitionFoo": []byte("\"foo\""),
```

Comment:
Just an FYI, go supports raw string literals via the backtick (`), so you could write this as:

```go
[]byte(`"foo"`)
```

I believe which makes the escaping a little less tedious.  Doesn't much matter here, but throwing it out there in case sometime in the future it helps you.

OTOH, you probably already knew about this since you use it in the multi-line string literal in the test :-).

Reply:
I'll update...  It looks better